design/FY2017/Final NFP-Single-Zone VAV Fan Control for Other Equipment Types.md

```diff
@@ -0,0 +1,710 @@
+
+Apparently, this aspect of the model fell short of perfection. Although the temperature limits are enforced, the fan coil air flow rate does not increase until the load threshold is exceeded. We may consider increasing supply air flow rate once the supply air temperature reaches the limits.<br>
+![](FanCoil_ASHRAE901_V86_SPc.png)<br>
+
+## Approach ##
+
+The team will:
+
+ - Investigate temperature limits using ASHRAE90.1 control method for ZoneHVAC:FourPipeFanCoil
+ - Add new ASHRAE90.1 control for AirloopHVAC:UnitarySystem using water coils.
+
+## Known Issues ##
+
+- Controlling supply air temperature **AND** air flow rate may prove troublesome
+
+## Testing/Validation/Data Sources ##
+
+Compare simulation results with existing equipment models. Document comparison of outlet temperature and humidity ratio and power consumption. 
+
+## Input Description ##
+
+The previous implementation for the Fan Coil model required four key inputs.
+
+ - Capacity Control
+ - low speed fan ratio
+ - Maximum Supply Air Temperature in Heating Mode
+ - Minimum Supply Air Temperature in Cooling Mode
+
+Reusing similar inputs in the Unitary System object is desirable as long as those inputs are not ambiguous. 
+
+ - For Capacity Control, an input is already available as Control Type
+ - For low speed fan ratio, the input for no load supply air flow rate could be used
+ - For Maximum Supply Air Temperature in Heating Mode use input for Maximum Supply Air Temperature
+ - For Minimum Supply Air Temperature in Cooling Mode one input is available but is ambiguous as currently named.
+   - DOAS DX Cooling Coil Leaving Minimum Air Temperature
+
+This field could be changed to Minimum Supply Air Temperature. Otherwise one or more new inputs are required to provide the same inputs as used in the Fan Coil model.
+
+## Outputs Description ##
+
+No change to output reporting
+
+## IDD - Input Data Dictionary ##
+
+Comments are added where fields are affected.
+
+    AirLoopHVAC:UnitarySystem,
+       \memo AirloopHVAC:UnitarySystem is a generic HVAC system type that allows any
+       \memo configuration of coils and/or fan. This object is a replacement of other
+       \memo AirloopHVAC objects. This object can be used in outdoor air systems,
+       \memo outdoor air units, air loops, and as zone equipment if desired.
+       \min-fields 14
+    A1,  \field Name
+       \required-field
+       \type alpha
+       \reference DOAToZonalUnit
+       \note Unique name for the Unitary System.
+    A2,  \field Control Type
+       \type choice
+       \key Load
+       \key SetPoint
+
+add new key here for capacity control method:
+
+       \key ASHRAE90VariableFan
+
+with description:
+
+       \default Load
+       \note Load control requires a Controlling Zone name.
+       \note SetPoint control requires set points at coil outlet node.
+       \note ASHRAE90VariableFan control requires a Controlling Zone name, water coils and two-speed or variable-speed fan
```

Comment:
Thanks for reviewing. I just got single-speed DX cooling and heating coils working. I'll update that note. I haven't tested multi-speed coils yet nor know if I have enough time to finish those coils (approaching 200 hr budget). Will still allow any coils in UnitarySystem but only apply the ASHRAE 90.1 methodology to coils I have tested. I've also revised these notes so some will change.